### PR TITLE
Start the ramp after setpoint is sent

### PR DESCRIPTION
**Pull request by KathrynBaker**
### Description of work

Need to send the start command when setting the setpoint on the 8800

### To test

Relates to https://github.com/ISISComputingGroup/IBEX/issues/1623

### Acceptance criteria

Difficult to test, but is in use on NDEMUONFE

---

#### Code Review

- [x] Is the code of an acceptable quality?
- [x] Are the PVs named according to the [naming standards](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/PV-Naming).
- [x] Have suitable [disable records](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Disable-records) been added?
- [x] Have suitable [simulation records](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Record-Simulation) been added?
- [x] Have the [finishing touches](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/IOC-Finishing-Touches) been applied?
- [ ] Is there an [emulator](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Emulating-Devices) for the device? - There isn't an existing emulator, as this is a bug fix and an emulator would be rather complex I hven't created one as part of this ticket
- [ ] Have the changes been documented in the [release notes](https://github.com/ISISComputingGroup/IBEX/wiki/ReleaseNotes_Dev). If so, do they describe the changes appropriately?

### Functional Tests

- [x] Do changes function as described? Add comments below that describe the tests performed.
- [x] Does the IOC respond correctly both in full and simulation mode, where it's possible to test both?
- [x] If there are multiple _0n IOCs, do they run correctly?

### Final steps

- [ ] Reviewer has updated the submodule in the main EPICS repo? See **Reviewing work for the subModules of EPICS** in the [Git workflow](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Git-workflow) page for details.
- [ ] Reviewer has moved the [release notes](https://github.com/ISISComputingGroup/IBEX/wiki/ReleaseNotes_Dev) entry for this ticket in the "Changes merged into master" section

### CODE REVIEW

**Comment by KathrynBaker** [comment]
Have removed the forward link so merging this should be safe enough

File: DFKPS/DFKPS-IOC-01App/Db/DFKPS_8800_status.db [hunk 1]
```diff
@@ -339,3 +339,19 @@ record(ao, "$(P)RAW:SP")
...
     info(autosaveFields, "VAL")
 }
+
+record(bo, "$(P)START")
+{
+    field(DESC, "Reset")
+    field(DTYP, "stream")
+    field(OUT, "@dfkps.proto start $(port) 0")
+    field(ZNAM, "Start")
+    field(ONAM, "Start")
```
Comment (John-Holt-Tessella) [inline DFKPS/DFKPS-IOC-01App/Db/DFKPS_8800_status.db]: Should these be the same both ZNAM and 0ZAME are Start.

Comment (KathrynBaker) [inline DFKPS/DFKPS-IOC-01App/Db/DFKPS_8800_status.db]: Yes, as with the reset records we tend to get.

**Comment by KathrynBaker** [comment]
This is actually the expected behaviour now. Further testing showed some continuing oddities with auto starting the ramp (not going to the correct set point), but never sending a start seemed to have other problems. By keeping the commands separate then it allows for a start to be sent (manually) but doesn't force the start to occur every time.

**Pull request merged**